lib/thrift/parser.ex

```diff
@@ -9,6 +9,9 @@ defmodule Thrift.Parser do
   @typedoc "A Thrift IDL line number"
   @type line :: pos_integer | nil
 
+  @typedoc "A map of Thrift IDL annotations"
```

Comment:
After looking at the .thrift file, it's a little more clear what "a map of Thrift IDL annotations" means, but at first this was confusing (a map from what to what?).  Maybe something like "A map of thrift annotation keys to values"?  Are they always in that format?